src/test/java/com/tdd/TddTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Robert Bosch GmbH. All rights reserved.
+ */
+package com.tdd;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Test;
+
+/**
+ * @author MSP5COB
+ *
+ */
+public class TddTest {
+
+  @Test
+  public void test1() {

```

Comment:
Use the name of the test function to express functionality. E.g., `test1` can be `twoConsecutiveIntegersMakeOneRange`
That way, it will be easy to check if some functionality is left out.